core/bind/core_bind.cpp

```diff
@@ -2090,21 +2090,7 @@ PoolVector<uint8_t> _File::get_buffer(int64_t p_length) const {
 
 String _File::get_as_text() const {
 	ERR_FAIL_COND_V_MSG(!f, String(), "File must be opened before use.");
-
-	String text;
-	uint64_t original_pos = f->get_position();
-	f->seek(0);
-
-	String l = get_line();
-	while (!eof_reached()) {
-		text += l + "\n";
-		l = get_line();
-	}
-	text += l;
-
-	f->seek(original_pos);
-
-	return text;
+	return FileAccess::get_file_as_string(f->get_path_absolute());
```

Comment:
This will cause reopening the file which is already open, not sure if that's a good idea.

Reply:
Fixed!